doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -1,383 +1,522 @@
+
+.. image:: predefined_poses/setup_assistant_panda_predefined_hand_open_pose.png
+   :width: 700px
+
+* Add a ``close`` pose for the ``hand`` group with joint value ``0.0``.
+
+.. image:: predefined_poses/setup_assistant_panda_predefined_hand_close_pose.png
+   :width: 700px
+
+Note: Only ``panda_finger_joint1`` appears in the list as ``panda_finger_joint2`` mimics its values.
+
+After the previous steps, the following set of robot poses should be defined for the ``panda_arm`` and ``hand`` groups.
+
+.. image:: predefined_poses/setup_assistant_panda_predefined_poses_done.png
    :width: 700px
 
 Step 6: Label End Effectors
 ---------------------------
 
-We have already added the gripper of the Panda. Now, we
-will designate this group as a special group:
-**end effectors**. Designating this group as end effectors allows
-some special operations to happen on them internally.
+Now that we have added the hand of the Panda as a move group, we can designate it
+as an end effector. By designating a group as an end effector, MoveIt can perform certain special operations on it.
+For example, end effectors can be used for attaching objects to the arm while carrying out pick-and-place tasks.
 
-* Click on the *End Effectors* pane.
+* Click on the **End Effectors** pane.
 
-* Click *Add End Effector*.
+* Click **Add End Effector**.
 
-* Choose **hand** as the *End Effector Name* for the gripper.
+* Choose ``hand`` as the **End Effector Name** for the gripper.
 
-* Select **hand** as the *End Effector Group*.
+* Select ``hand`` as the **End Effector Group**.
 
-* Select **panda_link8** as the *Parent Link* for this end-effector.
+* Select ``panda_link8`` as the **Parent Link** for this end-effector.
 
-* Leave *Parent Group* blank.
+* Leave **Parent Group** blank.
 
-.. image:: setup_assistant_panda_end_effector_add.png
+.. image:: setup_assistant_panda_add_end_effector.png
    :width: 700px
 
 Step 7: Add Passive Joints
 --------------------------
 
-The passive joints tab is meant to allow specification of any passive
-joints that might exist in a robot. These are joints that are unactuated
-on a robot (e.g. passive casters.) This tells the planners that they
-cannot (kinematically) plan for these joints because they can't be
-directly controlled. The Panda does not have any passive
-joints so we will skip this step.
+The **Passive Joints** pane is meant to allow specification of any passive
+joints that might exist in a robot. These are joints that are unactuated,
+meaning that they cannot be directly controlled. It's important to specify
+passive joints so that the planners are aware of their existence and can avoid
+planning for them. If the planners do not know about the passive joints, they
+might try to plan trajectories that involve moving the passive joints, which would
+result in invalid plans. The Panda robot arm does not have any passive joints so we will skip this step.
 
+Step 8: ros2_control URDF Modification
```

Comment:
Suggest adding a RST `.. note::` here that lets the user know if their robots URDF/xacro already has a ros2_control.xacro included that this step can be skipped.

Reply:
Thank you for pointing that out. It will help prevent a lot of confusion.